katas/mumbling.js

```diff
@@ -0,0 +1,12 @@
+function mumbling(str) {
+  if (!isNaN(str)) throw new Error("no numbers allowed");
+  const a = str.toUpperCase().split("");
+  const timesIdx = a.reduce((acc, curr, currIdx) => {
+    const wildcard = "-";
```

Comment:
wildcard could be defined before 'const a'. In other words, outside the reduce callback. This way you are defining wildcard with every element of the array 'a'.

Reply:
cool! cheers!